test/codeu/chat/server/storage/StorageTest.java

```diff
@@ -49,15 +45,71 @@ public void setup(){
 		}
 	}
 
+	//Resets fields for next tests
+	private void reset() throws SQLException{
+		storage.messageTable.destroy();
+		storage.conversationTable.destroy();
+		storage = new Storage(database);
+		conversations.clear();
+	}
+
+	@Before
+	public void setup(){
+		database = new Database("test.db");
+		username = "test";
+		username2 = "test2";
+		conversations = new ArrayList<ConversationData>();
+		storage = new Storage(database);
+	}
+
+	@Test
+	public void testAddingData() throws SQLException{
+		addToStorage(15, 3, username);
+		addToStorage(12, 3, username2);
+		List<DBObject<ConversationSchema>> addedConvos = storage.conversationTable.find(ConversationSchema.UNIQUE_ID, username);
+		assertEquals(addedConvos.size(), 15);
```

Comment:
You will also need to verify the data that you just read to confirm it matches with the expected value.